Android/Yash Raj Gupta/Advanced/NewsApp/app/src/main/java/com/example/newsapp/view/NewsDetailsActivity.java

```diff
@@ -0,0 +1,111 @@
+package com.example.newsapp.view;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.Toast;
+
+import com.example.newsapp.R;
+
+import androidx.appcompat.app.ActionBar;
+import dagger.android.support.DaggerAppCompatActivity;
+
+public class NewsDetailsActivity extends DaggerAppCompatActivity {
```

Comment:
NavigationDrawer implementation is missed out..Please check
Navigation drawer can contain the diff news categories like Sports, Politics, Business etc.